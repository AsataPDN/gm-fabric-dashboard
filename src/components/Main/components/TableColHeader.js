import TableCol from "./TableCol";

import { spacingScale, rowChildSpacing } from "style/styleFunctions";

const TableColHeader = TableCol.extend`
  ${props => (props.paddingLeft ? `padding-left: ${spacingScale(2)};` : "")};
  ${props => (props.paddingRight ? `padding-right: ${spacingScale(2)};` : "")};
  ${rowChildSpacing()};
`;
export default TableColHeader;
