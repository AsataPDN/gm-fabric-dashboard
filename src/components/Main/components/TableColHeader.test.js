import React from "react";
import { shallow } from "enzyme";
import TableColHeader from "./TableColHeader";

describe("TableColHeader", () => {
  it("matches snapshot", () => {
    const aTableColHeader = shallow(<TableColHeader />);
    expect(aTableColHeader).toMatchSnapshot();
  });
});
