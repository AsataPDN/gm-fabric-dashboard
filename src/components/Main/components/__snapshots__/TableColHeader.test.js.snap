// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableColHeader matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 12%;
  -ms-flex: 1 1 12%;
  flex: 1 1 12%;
  text-align: left;
  padding-right: 16px;
  padding-top: 7px;
  padding-bottom: 7px;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

.c0:first-child {
  padding-left: 16px;
}

<div
  className="c0"
/>
`;
